src/test/kotlin/no/nav/helse/flex/client/bregDirect/EnhetsregisterClientTest.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.helse.flex.client.bregDirect
+
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.web.client.RestClient
+
+class EnhetsregisterClientTest {
+    private lateinit var client: EnhetsregisterClient
+
+    private val mockWebServer: MockWebServer =
+        MockWebServer().apply {}
+
+    @BeforeEach
+    fun setUp() {
+        mockWebServer.start()
+        val baseUrl = mockWebServer.url("/").toString().removeSuffix("/")
+        val rc =
+            RestClient
+                .builder()
+                .baseUrl(baseUrl)
+                .build()
+        client = EnhetsregisterClient(enhetsregisterRestClient = rc)
+    }
+
+    @AfterEach
+    fun tearDown() {
+        mockWebServer.shutdown()
+    }
+
+    @Test
+    fun `erDagmamma returnerer NEI for ikke-dagmamma org`() {
+        val json = """{"naeringskode1": {"kode": "41.109"}}"""
+        mockWebServer.enqueue(MockResponse().setBody(json).setHeader("Content-Type", "application/json"))
+
+        val status = client.erDagmamma("509100675")
+        assertEquals(false, status)
```

Comment:
assertTrue(erDagmma)? eller bruk Kluent, selv om det ikke git all verdens i dette tilfelle: 'status.`should be true`()'